src/assets/scss/_intro-section.scss

```diff
@@ -11,7 +11,7 @@
     padding-bottom: 150px;
 
     .touch & {
-      background-image: url(../images/classroom.jpg);
+      background-image: url("http://www.aspaces.ca/wp-content/uploads/2014/12/20141218_135413-600x300.jpg");
```

Comment:
it's considered bad practice to use someone else's URL because it will eat off their server's resources. instead, you should have a copy of the photo committed as part of this pull request.